telegram/ext/_dispatcher.py

```diff
@@ -631,6 +632,49 @@ def remove_handler(self, handler: Handler, group: int = DEFAULT_GROUP) -> None:
             if not self.handlers[group]:
                 del self.handlers[group]
 
+    def migrate_chat_data(
+        self, message: 'Message' = None, old_chat_id: int = None, new_chat_id: int = None
+    ) -> None:
+        """Moves the contents of :attr:`chat_data` at key old_chat_id to the key new_chat_id.
+        Also updates the persistence by calling :attr:`update_persistence`.
+
+        Warning:
+
+            * Any data stored in :attr:`chat_data` at key `new_chat_id` will be overridden
+            * The key `old_chat_id` of :attr:`chat_data` will be deleted
+
+        Args:
+            message (:class:`Message`, optional): A message with either
+                :attr:`telegram.Message.migrate_from_chat_id` or
+                :attr:`telegram.Message.migrate_to_chat_id`.
```

Comment:
Just an idea that I get while re-reading this: We could rename the parameters `old/new_chat_id` to `migrate_from_to_chat_id` to more closely match this naming. @Poolitzer what do you think?

Reply:
@harshil21 do you have an opinion on this?

Reply:
Hm, I think the shorter names make it directly clear what each parameter does, I like them. I can also see the point of staying closer to the function name, though. Hard decision to do.

I am a bit more on the side for the shorter names though.

Reply:
totally missed this - Yeah, I prefer old/new chat id too

Reply:
all right, then let's keep it :)